libs/bulldozer/application/features/navigation/src/lib/navigation.component.ts

```diff
@@ -104,8 +105,26 @@ import { DarkThemeService } from '@heavy-duty/bulldozer/application/ui/dark-them
         width: 200px;
       }
 
-      .sidenav .mat-toolbar {
-        background: inherit;
+      .mat-toolbar.mat-primary {
```

Comment:
are these styles needed? I think they're not needed after moving the code editor

Reply:
All those styles related to the edit